woocommerce/woocommerce-android
# Init Connect and Scan fragments + VMs

Parent issue #3725 


This PR introduces skeletons for "CardReaderConnectFragment" a "CardReaderScanFragment" and their VMs. 


"CardReaderConnectFragment" - corresponds to "Connect reader" screen. The code in this fragment and its VM should follow the best practices. ([designs](https://user-images.githubusercontent.com/2261188/112047373-d5f1d500-8b12-11eb-9c44-5f4607524ee8.png))
"CardReaderScanFragment" - corresponds to "SeveralReadersFound" screen. The code in this fragment and its VM should follow the best practices. ([designs](https://user-images.githubusercontent.com/2261188/112047373-d5f1d500-8b12-11eb-9c44-5f4607524ee8.png))
"CardReaderSettingsFragment" (not introduced in this PR) - we plan to either remove this fragment or use it just as an entry point into "connect to reader" flow. The code in this fragment is just temporary and doesn't need to be reviewed.


To Test:
1. Open app settings
2. Tap on "Manage card readers"
3. Tap on "Redirect to ConnectFragment"
4. Notice the app displays "ConnectFragment"
5. Tap on "Initiate Scan"
6. Notice the app displays "ScanFragment

Update release notes:
